modules/user_accounts/php/module.class.inc

```diff
@@ -35,24 +35,8 @@ class Module extends \Module
      */
     public function hasAccess(\User $user) : bool
     {
-        return parent::hasAccess($user) && $user->hasPermission('user_accounts');
-    }
-
-    /**
-     * The edit_user page overrides the default handler in order to delegate
-     * edit_user/$identifeir to the edit_user page.
-     *
-     * @param ServerRequestInterface $request The incoming PSR7 request
-     *
-     * @return ResponseInterface The outgoing PSR15 response
-     */
-    public function handle(ServerRequestInterface $request) : ResponseInterface
```

Comment:
I don't understand why this is being moved to the useraccounts page or what it has to do with data frameworkification.

Reply:
Hmm, I was just following the general template of other dataframeworkified modules that don't have a `handle` function in the `module` class, and instead have it in the `endpoint` class. Should I leave it in the module class instead?

Reply:
yes, Modules have handle functions, they just aren't always overloaded. Moving this changes the logic.

Reply:
okay!